client/state/signup/progress/actions.js

```diff
@@ -97,6 +97,7 @@ function recordSubmitStep( flow, stepName, providedDependencies, optionalProps )
 	);
 
 	const device = resolveDeviceTypeByViewPort();
+
```

Comment:
`Non-blocking / Nit`:

Not a big deal at all, but we introduced a newline here